### ISSUE TITLE
Is there a way to monitor pre-training ? 

### ISSUE THREAD

**Issue by MoMe36** [Unity-Technologies/ml-agents#2492]
Hi ! 

I'm training a PPO agent. To take advantage of some demonstrations I made, I'm relying on pre-training + GAIL. On launching the training session, the terminal starts by indicating progresses in terms of RL (mean reward per episode). Is there a way I can monitor the pre-training (loss over epochs, for instance). As of now, I am not even sure it is doing the pretraining. 
Here is my config: 
```
default:
    trainer: ppo
    batch_size: 256
    beta: 5.0e-3
    buffer_size: 2560
    epsilon: 0.2
    hidden_units: 128
    lambd: 0.95
    learning_rate: 3.0e-4
    max_steps: 1.0e6
    memory_size: 256
    normalize: false
    num_epoch: 3
    num_layers: 2
    time_horizon: 64
    sequence_length: 64
    summary_freq: 1000
    use_recurrent: false
    pretraining:
        demo_path: ./demos/PickAndPlaceHCL.demo
        strength: 0.5
        steps: 10000
    reward_signals:
        extrinsic:
            strength: 1.0
            gamma: 0.99
        gail:
            strength: 0.03
            gamma: 0.99
            encoding_size: 128
            demo_path: demos/PickAndPlaceHCL.demo

```
The training terminal, showing no informations about pre-training

![Capture du 2019-09-04 14-34-53](https://user-images.githubusercontent.com/18698421/64255307-6365d800-cf21-11e9-9001-e2f5fbe085a1.png)

Thanks a lot !

**Comment by ervteng** [comment]
Hi @MoMe36, the pretraining is happening at the same time as the RL (it's better for the value network) and anneals over time. If you view the Tensorboard, you can see the pretraining loss and learning rate decreasing, and 0 out when pretraining `steps` is reached.

**Comment by MoMe36** [comment]
Oh, okay, I see. I was actually thinking that pre-training was a stage on its own. 
Thanks ! 

Closing the issue.
